keras_hub/src/models/cspnet/cspnet_backbone.py

```diff
@@ -0,0 +1,1269 @@
+import keras
+from keras import layers
+from keras import ops
+
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.feature_pyramid_backbone import FeaturePyramidBackbone
+from keras_hub.src.utils.keras_utils import standardize_data_format
+
+
+@keras_hub_export("keras_hub.models.CSPNetBackbone")
+class CSPNetBackbone(FeaturePyramidBackbone):
+    """This class represents Keras Backbone of CSPNet model.
+
+    This class implements a CSPNet backbone as described in
+    [CSPNet: A New Backbone that can Enhance Learning Capability of CNN](
+        https://arxiv.org/abs/1911.11929).
+
+    Args:
+        stem_filters: int or list of ints, filter size for the stem.
+        stem_kernel_size: int or tuple/list of 2 integers, kernel size for the
+            stem.
+        stem_strides: int or tuple/list of 2 integers, stride length of the
+            convolution for the stem.
+        stackwise_num_filters: A list of ints, filter size for each block level
+            in the model.
+        stackwise_strides: int or tuple/list of ints, strides for each block
+            level in the model.
+        stackwise_depth: A list of ints, representing the depth
+            (number of blocks) for each block level in the model.
+        block_type: str. One of `"bottleneck_block"`, `"dark_block"`, or
+            `"edge_block"`. Use `"dark_block"` for DarkNet blocks,
+            `"edge_block"` for EdgeResidual / Fused-MBConv blocks.
+        groups: int, specifying the number of groups into which the input is
```

Comment:
rename to num_groups

Reply:
This is the standard groups argument which we follow in Conv2D layers.